### ISSUE TITLE
invalid URL

### ISSUE THREAD

**Issue by jayhesselberth** [rnabioco/clustifyr#393]
I'm seeing an invalid URL in the docs. Did we set this up? Or is this a public resource?

```r
> urlchecker::url_check()
✖ Error: man/sce_small.Rd:11:6 : Invalid URI scheme
\url{"https://scrnaseq-public-datasets.s3.amazonaws.com/scater-objects/segerstolpe.rds"}
```

**Comment by kriemo** [comment]
It's from here, but the dataset download link is now invalid:
https://hemberg-lab.github.io/scRNA.seq.datasets/human/pancreas/#segerstolpe

**Comment by kriemo** [comment]
There is R code to document how it was made that we could reference:
https://github.com/hemberg-lab/scRNA.seq.datasets/blob/master/R/segerstolpe.R

**Comment by jayhesselberth** [comment]
closed by 1131c12961a4d0965e9780ff28c211c07ab79f7e
